eulercrit.py

```diff
@@ -10,12 +10,31 @@ def eulerCrit(a,p):
         print "[INFO] {} sayisi, {} sayisi icin quadratic residue degildir!".format(a, p)
 
 if __name__ == "__main__":
-    p=int(argv[1])
-    a=int(argv[2])
     if len(argv)<3:
-        print "[ERR] Eksik argüman girildi!"
-        print "[INFO] Doğru kullanım : python eulercrit.py <test edilecek asal sayi> <test edilecek quadratic residue>"
+        print "[ERR] Eksik parametreler var!"
+        print "[INFO] Doğru kullanım : python eulercrit.py <asal moduler ust sinir> <quadratic residue>"
+        print "[INFO] Istege Bagli : python eulercrit.py [--list|-l] <moduler ust sinir>"
+        exit(1)
+    if (argv[1]=="--list" or argv[1]=="-l"):
```

Comment:
python eulercrit.py 2244252 -l 5 formatında bir argüman beslemesi burayı hataya düşürür.Bunun önüne geçmek için -l|--list argümanıyla ardından gelen argümanı birbirine bağlaman gerek.